content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,255 @@ title: Instrumentation
 weight: 3
 ---
 
-This guide will cover creating and annotating spans, creating and annotating metrics, how to pass context, and a guide to automatic instrumentation for JavaScript. This simple example works in the browser as well as with Node.js
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+If a `TracerProvider` is not created, the OpenTelemetry APIs for tracing will
+use a no-op implementation and fail to generate data.
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+### Node.js
 
-function doWork() {
-  console.log('work...');
-  // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
-}
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/resources \
+  @opentelemetry/semantic-conventions \
+  @opentelemetry/sdk-trace-node \
+  @opentelemetry/instrumentation
+```
+
+Next, create a separate `tracing.js` file that has all the SDK initialization
+code in it:
+
+```js
+const opentelemetry = require("@opentelemetry/api");
+const { Resource } = require("@opentelemetry/resources");
+const { SemanticResourceAttributes } = require("@opentelemetry/semantic-conventions");
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { ConsoleSpanExporter, BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
+
+const resource =
+  Resource.default().merge(
+    new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: "service-name-here",
+      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
+    })
+  )
+
+const provider = new NodeTracerProvider({
+    resource: resource,
+});
+const exporter = new ConsoleSpanExporter();
+const processor = new BatchSpanProcessor(exporter);
+provider.addSpanProcessor(processor);
+
+// Optionally register automatic instrumentation libraries
+registerInstrumentations({
```

Comment:
i would move the `registerInstrumentations` before the initialization of the exporter (L55) since 3rd party exporter might require dependency before the instrumentation can patch it (specially http)